playwright-ct-angular/package.json

```diff
@@ -40,18 +40,20 @@
     }
   },
   "dependencies": {
+    "@analogjs/vite-plugin-angular": "0.2.0-beta.1",
     "@angular/common": "^15.2.0",
     "@angular/compiler": "^15.2.0",
     "@angular/core": "^15.2.0",
     "@angular/platform-browser": "^15.2.0",
     "@angular/platform-browser-dynamic": "^15.2.0",
     "@angular/router": "^15.2.0",
     "rxjs": "~7.8.0",
-    "tslib": "^2.5.0",
+    "tslib": "^2.4.0",
     "vite": "^4.1.4",
-    "zone.js": "~0.12.0"
+    "zone.js": "~0.13.0"
   },
   "devDependencies": {
+    "@angular-devkit/build-angular": "^16.0.0-rc.0",
```

Comment:
Can probably removed when: https://github.com/analogjs/analog/issues/344 is resolved